### PR TITLE
Jaeger bazel (Linux only)

### PR DESCRIPTION
**Pull request by esigo**
Jaeger bazel support

## Changes
Adds Jaeger bazel support.
Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

**Comment by lalitb** [comment]
Sorry for the delay in reviewing this. would try to review it by today.

File: WORKSPACE [hunk 1]
```diff
@@ -36,5 +36,35 @@ upb_deps()
...
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_foreign_cc",
+    sha256 = "69023642d5781c68911beda769f91fcbc8ca48711db935a75da7f6536b65047f",
+    strip_prefix = "rules_foreign_cc-0.6.0",
+    url = "https://github.com/bazelbuild/rules_foreign_cc/archive/0.6.0.tar.gz",
+)
+
+load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
```
Comment (lalitb) [inline WORKSPACE:48]: Nit: Can we use `rules_foreign_cc` to install boost library too - if that helps to build for Windows. Probably, we can create a github issue for Windows support, for someone to pick it up.

**Review by lalitb — APPROVED** [review]
Thanks. Looks good.

File: WORKSPACE (hunk 1 above)
Comment (esigo) [reply WORKSPACE:48]: Sure, I'll raise another PR for Windows support.

**Pull request merged**